src/operator/controllers/secrets/types/secret_types.go

```diff
@@ -0,0 +1,104 @@
+package secretstypes
+
+import (
+	"context"
+	"github.com/samber/lo"
+	"strings"
+)
+
+type SecretType string
+type CertType string
+
+const (
+	TlsSecretType = SecretType("TLS")
+	JksCertType   = CertType("jks")
+	PemCertType   = CertType("pem")
+)
+
+func StrToCertType(strCertType string) CertType {
+	switch CertType(strings.ToLower(strCertType)) {
+	case JksCertType:
+		return JksCertType
+	case PemCertType:
+		return PemCertType
+	default:
+		return PemCertType
+	}
+}
+
+type CertificateData struct {
+	Files     map[string][]byte
+	ExpiryStr string
+}
+
+type PemConfig struct {
+	SvidFileName   string
+	BundleFileName string
+	KeyFileName    string
+}
+
+func NewPemConfig(svidFileName string, bundleFileName string, keyFileName string) PemConfig {
+	newFileNames := PemConfig{}
+	newFileNames.SvidFileName, _ = lo.Coalesce(svidFileName, "svid.pem")
+	newFileNames.KeyFileName, _ = lo.Coalesce(keyFileName, "key.pem")
+	newFileNames.BundleFileName, _ = lo.Coalesce(bundleFileName, "bundle.pem")
+	return newFileNames
+}
+
+type JksConfig struct {
```

Comment:
Change the acronyms here to be all uppercase - JKS, PEM, SVID, etc. The go linter recommends this change - it's part of the style guide. Basically, the rule is to write like you would as part of written text - if you would write JKS, SVID, HTTP, URL, etc, then you should do the same in variable names.